### PR TITLE
Switched keys and values

### PR DESCRIPTION
**Pull request by artistmatej**
Now the keys are on the left side and the values on the right. (Resolve https://github.com/pyvista/pyvista/issues/541)

### CODE REVIEW

**Comment by banesullivan** [comment]
Thanks for taking this on, @artistmatej! I will take a look at this soon and adjust the failing test if needed (I think the test needs to be adjusted for these changes)

**Comment by banesullivan** [comment]
Also, it would be great to update all of the example outputs in the README (including the screenshot)

File: scooby/report.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def cols(html, version, name, ncol, i):
         # Loop over packages
         i = 0  # Reset count for rows.
         for name, version in self.packages.items():
-            html, i = cols(html, version, name, self.ncol, i)
+            html, i = cols(html, name, version, self.ncol, i)
```
Comment (banesullivan) [inline scooby/report.py]: Actually we don't want to do any of these changes but switch where the variables are used **inside** the `cols` method. These changes pass the `name` as the `version` argument and `version` as the `name` argument leading to a significant inconsistency. Revert this change and change the internals of the `cols` method to the following:

```py
def cols(html, version, name, ncol, i):
            r"""Print package information in two cells."""

            # Check if we have to start a new row
            if i > 0 and i % ncol == 0:
                html += "  </tr>\n"
                html += "  <tr>\n"

            html += "    <td style='text-align: right; background-color: #ccc;"
            html += " " + border + ">%s</td>\n" % name

            html += "    <td style='text-align: left; "
            html += border + ">%s</td>\n" % version

            return html, i+1
```

Comment (artistmatej) [reply scooby/report.py]: Ok. I'll do that. 

Another question. At the moment I left the space for RAM empty. Should I write RAM inside and write an ? or _ or something else as value? 

image

If I leave it as before, the edge is missing a border, like here:

image

Comment (artistmatej) [reply scooby/report.py]: Don't mind the yellow spot.

Comment (banesullivan) [reply scooby/report.py]: can you install `psutil` so Scooby will report on the RAM? It's an optional dependency

Comment (banesullivan) [reply scooby/report.py]: That edge is weird, though... not sure why that is happening

File: scooby/report.py [hunk 2]
```diff
@@ -283,6 +283,8 @@ def cols(html, version, name, ncol, i):
         html, i = cols(html, 'Architecture', self.architecture, self.ncol, i)
         if TOTAL_RAM:
             html, i = cols(html, 'RAM', self.total_ram, self.ncol, i)
+        else:
+            html, i = cols(html, '', '', 1, 0)
```
Comment (banesullivan) [inline scooby/report.py]: I don't think we want this. Other entries can be added to that section by the user and this would add an awkward blank entry when the RAM isn't reported

Comment (artistmatej) [reply scooby/report.py]: Now only Plain Text and HTML order are changed and nothing else.

Is this image good enough for the README ? Or is some data missing in compared to
the old image.

image


--------------------------------------------------------------------------------
  Date: Sat Feb 08 14:30:36 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 63.9 GB
       Environment : Jupyter

  Python 3.7.3 (v3.7.3:ef4ec6ed12, Mar 25 2019, 22:22:05) [MSC v.1916 64 bit
  (AMD64)]

             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.1
--------------------------------------------------------------------------------

Comment (artistmatej) [reply scooby/report.py]: This is how the reports look now and I could include them to README

Jupyter

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:45 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:56 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.23.1
               vtk : 8.1.2
        no_version : Could not import
    does_not_exist : Could not import
             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

Comment (banesullivan) [reply scooby/report.py]: That looks good!

File: scooby/report.py [hunk 3]
```diff
@@ -220,7 +220,7 @@ def __repr__(self):
 
         # Loop over packages
         for name, version in self._packages.items():
-            text += '{:>18} : {}\n'.format(version, name)
+            text += '{:>18} : {}\n'.format(name, version)
```
Comment (banesullivan) [inline scooby/report.py]: I'm wondering if this should be `{:>21}` to be consistent with the above section. Thoughts, @prisae?

Comment (banesullivan) [reply scooby/report.py]: Actually, the output visibly lines up when testing locally...

**Comment by artistmatej** [comment]
These are the reports from HTML and Plain Text. And if these are ok, where should I upload the image or do you upload the images to your repository?

This is how the reports look now and I could include them to README

Jupyter

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:45 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

```
--------------------------------------------------------------------------------
  Date: Mon Feb 10 09:00:56 2020 W. Europe Standard Time

                OS : Windows
            CPU(s) : 16
           Machine : AMD64
      Architecture : 64bit
               RAM : 31.9 GB
       Environment : IPython

  Python 3.7.6 | packaged by conda-forge | (default, Jan  7 2020, 21:48:41)
  [MSC v.1916 64 bit (AMD64)]

           pyvista : 0.23.1
               vtk : 8.1.2
        no_version : Could not import
    does_not_exist : Could not import
             numpy : 1.18.1
           IPython : 7.12.0
        matplotlib : 3.0.3
            scooby : 0.5.0

  Intel(R) Math Kernel Library Version 2019.0.4 Product Build 20190411 for
  Intel(R) 64 architecture applications
--------------------------------------------------------------------------------
```

**Comment by prisae** [comment]
I will do tomorrow!

File: scooby/report.py (hunk 3 above)
Comment (prisae) [reply scooby/report.py]: Yes, we should be consistent. But this here remains the same, the above should change.

File: scooby/report.py [hunk 4]
```diff
@@ -201,13 +201,13 @@ def __repr__(self):
         text += date_text+'\n'
 
         # ########## Platform/OS details ############
-        text += '{:>18}'.format(self.system)+' : OS\n'
-        text += '{:>18}'.format(self.cpu_count)+' : CPU(s)\n'
-        text += '{:>18}'.format(self.machine)+' : Machine\n'
-        text += '{:>18}'.format(self.architecture)+' : Architecture\n'
+        text += '{:>21}{}'.format('OS : ', self.system)+'\n'
```
Comment (prisae) [inline scooby/report.py]: Would you mind changing all (lines 204-207, 209, 210)
- `{:>21}{}` to `{:>18} : {}\n`
and then remove later in the line the
-  ` : `, and 
- `+'\n'`?
 This way it would all be consistent and nice. It would even be more consistent than before!

Thanks for your contribution!

Comment (prisae) [reply scooby/report.py]: (Basically change it to the format as used in `Loop over packages`.)

**Comment by prisae** [comment]
Thanks @artistmatej

**Comment by prisae** [comment]
@banesullivan from my point of view it can be merged. You wanted to update the README first, or within this PR?

**Comment by artistmatej** [comment]
I can update within this PR, I just don't know how to upload the image of Jupyter report. Right now it's in this folder https://github.com/banesullivan/scooby/raw/master/assets/jupyter.png to which I have no access. Or should I just drag and drop and it lands where it lands?

**Comment by prisae** [comment]
It is in the folder `assets` in your branch, you should just be able to replace the existing screenshot.
Selection_001

**Comment by artistmatej** [comment]
Ah, of course. Awesome! Thanks!

**Comment by artistmatej** [comment]
After all the changes we have made I get this report which is wrong:
        no_version : Could not import
    does_not_exist : Could not import

**Comment by prisae** [comment]
`no_version` is a dummy package, just to show what happens with a package which versioning is bad. You have to install it with `pip install no_version`. ;-)

**Comment by artistmatej** [comment]
Haha, that's a funny package. Thanks! Ok, I'll commit the README.md

**Comment by prisae** [comment]
Thanks @artistmatej !

**Comment by banesullivan** [comment]
Thanks @artistmatej and @prisae! Looks great!

**Comment by artistmatej** [comment]
You're welcome and thank you @banesullivan and @prisae for your help and input!
